src/App.vue

```diff
@@ -0,0 +1,294 @@
+<template>
+  <div>
+    <div class="focus">
+      <input type="text" ref="inputRef" />
+      <br /><br />
+      <div class="button-container">
+        <button @click="inputFieldFocus">Focus</button>
+        <button @click="toggleModal">Open Modal</button>
```

Comment:
create 2 button 
1 will send/set theme to sales another will remove it. SO we can test the theme functionality @Shashank-002